### DISCUSSION TITLE
Map Command+S shortcut to the data-model page?

### DISCUSSION THREAD

**Discussion by RSickenberg** [directus/directus#6581] [Feature Requests]
Hello, 

I realized there is a shortcut for saving with cmd+s but not everywhere, is it normal?

Regards;

**Comment by rijkvanzanten** [comment]
Good point, this should be the same everywhere 👍🏻

**Comment by licitdev** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
